### ISSUE TITLE
hicanu fail in cns step with "job ctgcns/00XX.cns FAILED"

### ISSUE THREAD

**Issue by fsq9451** [marbl/canu#1847]
Hi,
I was using ccs sequences for hicanu aasembly, here is details: 
canu \
-p miganda -d miganda_file \
genomeSize=400m \
-pacbio-hifi miganda.ccs.fastq.gz

However, I met this error and cannot be fixed by re-running it:
```
--Finished on Sat Nov 21 15:11:01 2020 (569 seconds) with 13687.742 GB free disk space
----------------------------------------
-- Using slow alignment for consensus (iteration '2').
-- Configured 17 consensus jobs.
--
-- Consensus jobs failed, tried 2 times, giving up.
--   job ctgcns/0001.cns FAILED.
--   job ctgcns/0002.cns FAILED.
--   job ctgcns/0003.cns FAILED.
--   job ctgcns/0004.cns FAILED.
--   job ctgcns/0005.cns FAILED.
--   job ctgcns/0006.cns FAILED.
--   job ctgcns/0007.cns FAILED.
--   job ctgcns/0008.cns FAILED.
--   job ctgcns/0009.cns FAILED.
--   job ctgcns/0010.cns FAILED.
--   job ctgcns/0011.cns FAILED.
--   job ctgcns/0012.cns FAILED.
--   job ctgcns/0013.cns FAILED.
--   job ctgcns/0014.cns FAILED.
--   job ctgcns/0015.cns FAILED.
--   job ctgcns/0016.cns FAILED.
--   job ctgcns/0017.cns FAILED.
--
ABORT:
ABORT: canu 2.1
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
```

I attached the log file here and need your help! Thanks
[nohup.out.log](https://github.com/marbl/canu/files/5577197/nohup.out.log)

**Comment by skoren** [comment]
Most likely this is the same as #1791 and #1799. You can check the error message in the logs for one of the failed jobs (`unitigging/5-*/*.out`). If it matches, update to version 2.1.1 which fixes the issue. If the error message looks different or 2.1.1 doesn't fix it, post the failing job log here.

**Comment by skoren** [comment]
Idle, most likely fixed by 2.1.1 release.
